src/resources/js/commerce/gateway/stripe/checkout.js

```diff
@@ -66,6 +67,15 @@ tribe.tickets.commerce.gateway.stripe.checkout = {};
 	 * @type {Object|null}
 	 */
 	obj.stripeElements = null;
+	
+	/**
+	 * Loader container.
+	 *
+	 * @since TBD
+	 *
+	 * @type {Object|null}
+	 */
+	obj.loaderContainer = null;
```

Comment:
For the loader we're showing, we're using the checkout container as the container. I think in order to be more clear this should be called `checkoutContainer`. But also, I'm not sure why is that we're saving this here rather than getting the container on the fly?

Reply:
This is to just avoid slow jQuery execution if the page has a lot of HTML elements. We are just looking for the same element each time, so caching it should not be a problem.